### ISSUE TITLE
Crash after "Check database"

### ISSUE THREAD

**Issue by Sunt-ing** [ankidroid/Anki-Android#7918]
## Reproduction Steps
1. Install AnkiDroid
2. Select "Check database"
3. Select "OK"

## Actual Result
Crashed.

## Debug Info
AnkiDroid Version = 2.15alpha17

Android Version = 9

ACRA UUID = 741efefa-e3a5-4eef-a2e8-e5032cead758

Scheduler = std

Crash Reports Enabled = true

Happened both on my `Honor` phone and the emulator `Pixel 3 API 28`.

## Demo
![ankidroid_crash](https://user-images.githubusercontent.com/43040147/102708364-929e8500-42dd-11eb-9d8d-049ff5012abe.gif)

**Comment by mikehardy** [comment]
Hi there!
Why would you use 2.14alpha18? We are on 2.15alpha now
Also, you do not include the crash UUID so we can't trace this
we can reopen if you reproduce this with the latest alpha and you include the UUID with crash reporting enabled so we can locate the stack trace to fix it

**Comment by Sunt-ing** [comment]
I pulled the latest updates from the master branch just now, and find the problem is still here.
Also, as you advised, I have shown all the debug info.

Please tell me what I still need to add if there is.
@mikehardy

**Comment by mikehardy** [comment]
Strange - I'm unable to see any crash reports for that user id

I think with from-source-code and alpha builds you may need to go into settings -> advanced and turn on crash reporting for it to send them - can you try that? then I can search for this ACRA UUID again and try to get the crash report

We turn off crash reports by default on source code builds because we expect crashes during development and don't want to send reports for them normally. You have to enable it every time the app restarts

**Comment by Sunt-ing** [comment]
1. After pulled from the master branch I tried reproducing it but failed. However, when I taped "Manage note types", it crashed again. This works both on my cell phone and my emulator. And I reported it.
2. All the debug info has been updated.
AnkiDroid Version = 2.15alpha17

Android Version = 9

ACRA UUID = 741efefa-e3a5-4eef-a2e8-e5032cead758

Scheduler = std

Crash Reports Enabled = true

### demo:

![3](https://user-images.githubusercontent.com/43040147/102781237-497c2d00-43d2-11eb-8a69-5855b87cb9a6.gif)

**Comment by mikehardy** [comment]
I continue to have no crash reports for this device. I must see the crash trace in order to debug it.
You are running an emulator, so you may capture (and paste in here) the crash reports yourself, that would be an incredible help.

If you could follow `adb logcat` while creating the crash and paste the trace we can probably fix it almost immediately

**Comment by Sunt-ing** [comment]
2020-12-21 22:04:31.302 13144-13144/com.ichi2.anki I/com.ichi2.anki: Not late-enabling -Xcheck:jni (already on)
2020-12-21 22:04:31.349 13144-13144/com.ichi2.anki W/com.ichi2.anki: Unexpected CPU variant for X86 using defaults: x86
2020-12-21 22:04:31.915 13144-13144/com.ichi2.anki I/ACRA: ACRA is disabled for com.ichi2.anki, initializing...
2020-12-21 22:04:31.978 13144-13144/com.ichi2.anki D/AnkiDroid: Startup - Application Start
2020-12-21 22:04:31.982 13144-13144/com.ichi2.anki I/UsageAnalytics: initialize()
2020-12-21 22:04:31.984 13144-13144/com.ichi2.anki D/UsageAnalytics: App tracking id 'tid' = UA-125800786-1
2020-12-21 22:04:32.013 13144-13144/com.ichi2.anki I/WebViewFactory: Loading com.android.chrome version 69.0.3497.100 (code 349710017)
2020-12-21 22:04:32.020 13144-13144/com.ichi2.anki I/com.ichi2.anki: The ClassLoaderContext is a special shared library.
2020-12-21 22:04:32.102 13144-13144/com.ichi2.anki I/cr_LibraryLoader: Time to load native libraries: 9 ms (timestamps 1505-1514)
2020-12-21 22:04:32.121 13144-13174/com.ichi2.anki E/cr_VariationsUtils: Failed reading seed file "/data/user/0/com.ichi2.anki/app_webview/variations_seed": /data/user/0/com.ichi2.anki/app_webview/variations_seed (No such file or directory)
2020-12-21 22:04:32.121 13144-13144/com.ichi2.anki I/chromium: [INFO:library_loader_hooks.cc(36)] Chromium logging enabled: level = 0, default verbosity = 0
2020-12-21 22:04:32.122 13144-13144/com.ichi2.anki I/cr_LibraryLoader: Expected native library version number "69.0.3497.100", actual native library version number "69.0.3497.100"
2020-12-21 22:04:32.138 13144-13175/com.ichi2.anki W/cr_ChildProcLH: Create a new ChildConnectionAllocator with package name = com.android.chrome, sandboxed = true
2020-12-21 22:04:32.148 13144-13144/com.ichi2.anki I/cr_BrowserStartup: Initializing chromium process, singleProcess=false
2020-12-21 22:04:32.305 13144-13144/com.ichi2.anki D/NetworkSecurityConfig: Using Network Security Config from resource network_security_config debugBuild: true
2020-12-21 22:04:32.322 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Ldalvik/system/CloseGuard;->get()Ldalvik/system/CloseGuard; (light greylist, reflection)
2020-12-21 22:04:32.322 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Ldalvik/system/CloseGuard;->open(Ljava/lang/String;)V (light greylist, reflection)
2020-12-21 22:04:32.322 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Ldalvik/system/CloseGuard;->warnIfOpen()V (light greylist, reflection)
2020-12-21 22:04:32.335 13144-13144/com.ichi2.anki I/GoogleAnalyticsImpl: Not participating in analytics sample (sample percentage vs random: 10 69)
2020-12-21 22:04:32.336 13144-13144/com.ichi2.anki I/UsageAnalytics: setOptIn(): from false to true
2020-12-21 22:04:32.336 13144-13144/com.ichi2.anki I/GoogleAnalyticsImpl: Not participating in analytics sample (sample percentage vs random: 10 83)
2020-12-21 22:04:32.336 13144-13144/com.ichi2.anki D/UsageAnalytics: setOptIn() optIn / sAnalytics.config().enabled(): true/true
2020-12-21 22:04:32.337 13144-13144/com.ichi2.anki I/UsageAnalytics: setDryRun(): true, warning dryRun is experimental
2020-12-21 22:04:32.346 13144-13144/com.ichi2.anki I/CompatV26: Creating notification channel with id/name: General Notifications/AnkiDroid
2020-12-21 22:04:32.354 13144-13144/com.ichi2.anki I/CompatV26: Creating notification channel with id/name: Synchronization/Synchronization
2020-12-21 22:04:32.359 13144-13144/com.ichi2.anki I/CompatV26: Creating notification channel with id/name: Global Reminders/Cards due
2020-12-21 22:04:32.363 13144-13144/com.ichi2.anki I/CompatV26: Creating notification channel with id/name: Deck Reminders/Reminders
2020-12-21 22:04:32.405 13144-13144/com.ichi2.anki I/AnkiDroidApp: AnkiDroidApp: Starting Services
2020-12-21 22:04:32.410 13144-13144/com.ichi2.anki I/CollectionHelper: Begin openCollection: /storage/emulated/0/AnkiDroid/collection.anki2
2020-12-21 22:04:32.475 13144-13144/com.ichi2.anki I/Collection: Opening Collection Log
2020-12-21 22:04:32.480 13144-13144/com.ichi2.anki D/Collection: [1608559472] Collection.java:<init>(): /storage/emulated/0/AnkiDroid/collection.anki2,  2.15alpha17
2020-12-21 22:04:32.497 13144-13144/com.ichi2.anki D/Collection: Reflectively discovered database default cursor window size 2097152
2020-12-21 22:04:32.526 13144-13144/com.ichi2.anki D/Collection: [1608559472] Sched.java:_updateCutoff(): 22,  1608609600
2020-12-21 22:04:32.528 13144-13144/com.ichi2.anki I/CollectionHelper: End openCollection: /storage/emulated/0/AnkiDroid/collection.anki2
2020-12-21 22:04:32.528 13144-13144/com.ichi2.anki I/BootService: Executing Boot Service
2020-12-21 22:04:32.569 13144-13144/com.ichi2.anki D/IntentHandler: onCreate()
2020-12-21 22:04:32.584 13144-13144/com.ichi2.anki V/IntentHandler: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 cmp=com.ichi2.anki/.IntentHandler }
2020-12-21 22:04:32.585 13144-13144/com.ichi2.anki D/IntentHandler: onCreate() performing default action
2020-12-21 22:04:32.585 13144-13144/com.ichi2.anki I/IntentHandler: Launching DeckPicker
2020-12-21 22:04:32.656 13144-13144/com.ichi2.anki W/ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@a82a762
2020-12-21 22:04:32.731 13144-13144/com.ichi2.anki D/AnkiDroidApp: AnkiDroidApp::getLanguageConfig - setting locale to en_US
2020-12-21 22:04:32.755 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/content/res/Resources$Theme;->rebase()V (dark greylist, reflection)
2020-12-21 22:04:32.756 13144-13144/com.ichi2.anki I/ResourcesCompat: Failed to retrieve rebase() method
    java.lang.NoSuchMethodException: rebase []
        at java.lang.Class.getMethod(Class.java:2068)
        at java.lang.Class.getDeclaredMethod(Class.java:2047)
        at androidx.core.content.res.ResourcesCompat$ThemeCompat$ImplApi23.rebase(ResourcesCompat.java:501)
        at androidx.core.content.res.ResourcesCompat$ThemeCompat.rebase(ResourcesCompat.java:477)
        at androidx.appcompat.app.AppCompatDelegateImpl.attachBaseContext2(AppCompatDelegateImpl.java:465)
        at androidx.appcompat.app.AppCompatActivity.attachBaseContext(AppCompatActivity.java:107)
        at com.ichi2.anki.AnkiActivity.attachBaseContext(AnkiActivity.java:90)
        at android.app.Activity.attach(Activity.java:7051)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2873)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2020-12-21 22:04:32.761 13144-13144/com.ichi2.anki D/DeckPicker: onCreate()
2020-12-21 22:04:32.762 13144-13144/com.ichi2.anki I/DeckPicker: User has permissions to access collection
2020-12-21 22:04:32.763 13144-13144/com.ichi2.anki I/AnkiActivity: AnkiActivity::onCreate - DeckPicker
2020-12-21 22:04:32.791 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/drawable/Drawable;->getOpticalInsets()Landroid/graphics/Insets; (light greylist, linking)
2020-12-21 22:04:32.791 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/graphics/Insets;->left:I (light greylist, linking)
2020-12-21 22:04:32.791 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/graphics/Insets;->right:I (light greylist, linking)
2020-12-21 22:04:32.792 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/graphics/Insets;->top:I (light greylist, linking)
2020-12-21 22:04:32.792 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/graphics/Insets;->bottom:I (light greylist, linking)
2020-12-21 22:04:32.883 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/view/View;->getAccessibilityDelegate()Landroid/view/View$AccessibilityDelegate; (light greylist, linking)
2020-12-21 22:04:32.903 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/view/View;->computeFitSystemWindows(Landroid/graphics/Rect;Landroid/graphics/Rect;)Z (light greylist, reflection)
2020-12-21 22:04:32.906 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/view/ViewGroup;->makeOptionalFitsSystemWindows()V (light greylist, reflection)
2020-12-21 22:04:32.993 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/widget/PopupWindow;->setEpicenterBounds(Landroid/graphics/Rect;)V (light greylist, linking)
2020-12-21 22:04:32.996 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/widget/PopupWindow;->setClipToScreenEnabled(Z)V (light greylist, reflection)
2020-12-21 22:04:33.300 13144-13144/com.ichi2.anki D/DeckPicker: No DeckPicker background preference
2020-12-21 22:04:33.309 13144-13144/com.ichi2.anki I/DeckPicker: colOpen: true
2020-12-21 22:04:33.315 13144-13144/com.ichi2.anki I/DeckPicker: No startup screens required
2020-12-21 22:04:33.318 13144-13144/com.ichi2.anki D/BackupManager: performBackup: No backup created. Last backup younger than 5 hours
2020-12-21 22:04:33.324 13144-13144/com.ichi2.anki I/AnkiActivity: AnkiActivity::onStart - DeckPicker
2020-12-21 22:04:33.355 13144-13144/com.ichi2.anki D/DeckPicker: onResume()
2020-12-21 22:04:33.359 13144-13144/com.ichi2.anki I/AnkiActivity: AnkiActivity::onResume - DeckPicker
2020-12-21 22:04:33.360 13144-13144/com.ichi2.anki D/UsageAnalytics: sendAnalyticsScreenView(): DeckPicker
2020-12-21 22:04:33.360 13144-13144/com.ichi2.anki D/UsageAnalytics: getOptIn() status: true
2020-12-21 22:04:33.372 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Lsun/misc/Unsafe;->theUnsafe:Lsun/misc/Unsafe; (light greylist, reflection)
2020-12-21 22:04:33.374 13144-13144/com.ichi2.anki D/DialogHandler: Reading persistent message
2020-12-21 22:04:33.379 13144-13144/com.ichi2.anki D/DeckPicker$UpdateDeckListListener: Refreshing deck list
2020-12-21 22:04:33.380 13144-13144/com.ichi2.anki D/DeckPicker$UpdateDeckListListener: Refreshing deck list
2020-12-21 22:04:33.381 13144-13213/com.ichi2.anki D/CollectionTask$LoadDeck: doInBackgroundLoadDeckCounts
2020-12-21 22:04:33.382 13144-13144/com.ichi2.anki D/OpenGLRenderer: HWUI GL Pipeline
2020-12-21 22:04:33.387 13144-13213/com.ichi2.anki D/CollectionTask$LoadDeckCounts: doInBackgroundLoadDeckCounts
2020-12-21 22:04:33.423 13144-13203/com.ichi2.anki D/CollectionTask: Waiting for com.ichi2.async.CollectionTask$LoadDeck@8a3287e to finish before starting class com.ichi2.async.CollectionTask$LoadCollectionComplete
2020-12-21 22:04:33.423 13144-13203/com.ichi2.anki D/CollectionTask: Finished waiting for com.ichi2.async.CollectionTask$LoadDeck@8a3287e to finish. Status= RUNNING
2020-12-21 22:04:33.431 13144-13144/com.ichi2.anki I/DeckPicker$UpdateDeckListListener: Updating deck list UI
2020-12-21 22:04:33.471 13144-13144/com.ichi2.anki D/DeckPicker$UpdateDeckListListener: Startup - Deck List UI Completed
2020-12-21 22:04:33.471 13144-13144/com.ichi2.anki D/CollectionTask: enabling garbage collection of mPreviousTask...
2020-12-21 22:04:33.472 13144-13176/com.ichi2.anki D/AnkiStatsTaskHandler$DeckPreviewStatistics: Starting DeckPreviewStatistics
2020-12-21 22:04:33.472 13144-13176/com.ichi2.anki D/AnkiStatsTaskHandler$DeckPreviewStatistics: DeckPreviewStatistics query: select count(), sum(time)/1000 from revlog where id > 1608523200000
2020-12-21 22:04:33.495 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/view/WindowInsets;->CONSUMED:Landroid/view/WindowInsets; (light greylist, reflection)
2020-12-21 22:04:33.496 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/Insets;->of(IIII)Landroid/graphics/Insets; (light greylist, linking)
2020-12-21 22:04:33.527 13144-13214/com.ichi2.anki D/HostConnection: HostConnection::get() New Host Connection established 0xe42f4ac0, tid 13214
2020-12-21 22:04:33.554 13144-13214/com.ichi2.anki D/HostConnection: HostComposition ext ANDROID_EMU_CHECKSUM_HELPER_v1 ANDROID_EMU_dma_v1 ANDROID_EMU_YUV420_888_to_NV21 ANDROID_EMU_YUV_Cache ANDROID_EMU_async_unmap_buffer GL_OES_vertex_array_object GL_KHR_texture_compression_astc_ldr ANDROID_EMU_gles_max_version_2 
2020-12-21 22:04:33.570 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/os/Trace;->TRACE_TAG_APP:J (light greylist, reflection)
2020-12-21 22:04:33.570 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/os/Trace;->isTagEnabled(J)Z (light greylist, reflection)
2020-12-21 22:04:33.570 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/os/Trace;->asyncTraceBegin(JLjava/lang/String;I)V (light greylist, reflection)
2020-12-21 22:04:33.570 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/os/Trace;->asyncTraceEnd(JLjava/lang/String;I)V (light greylist, reflection)
2020-12-21 22:04:33.570 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/os/Trace;->traceCounter(JLjava/lang/String;I)V (light greylist, reflection)
2020-12-21 22:04:33.608 13144-13214/com.ichi2.anki I/ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
2020-12-21 22:04:33.609 13144-13214/com.ichi2.anki I/ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
2020-12-21 22:04:33.609 13144-13214/com.ichi2.anki I/OpenGLRenderer: Initialized EGL, version 1.4
2020-12-21 22:04:33.609 13144-13214/com.ichi2.anki D/OpenGLRenderer: Swap behavior 1
2020-12-21 22:04:33.609 13144-13214/com.ichi2.anki W/OpenGLRenderer: Failed to choose config with EGL_SWAP_BEHAVIOR_PRESERVED, retrying without...
2020-12-21 22:04:33.609 13144-13214/com.ichi2.anki D/OpenGLRenderer: Swap behavior 0
2020-12-21 22:04:33.634 13144-13214/com.ichi2.anki D/eglCodecCommon: setVertexArrayObject: set vao to 0 (0) 0 0
2020-12-21 22:04:33.634 13144-13214/com.ichi2.anki D/EGL_emulation: eglCreateContext: 0xe36b4f40: maj 2 min 0 rcv 2
2020-12-21 22:04:33.646 13144-13214/com.ichi2.anki D/EGL_emulation: eglMakeCurrent: 0xe36b4f40: ver 2 0 (tinfo 0xd1047880)
2020-12-21 22:04:33.694 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/drawable/DrawableContainer;->getOpticalInsets()Landroid/graphics/Insets; (light greylist, reflection)
2020-12-21 22:04:33.727 13144-13214/com.ichi2.anki D/HostConnection: createUnique: call
2020-12-21 22:04:33.727 13144-13214/com.ichi2.anki D/HostConnection: HostConnection::get() New Host Connection established 0xd07fcae0, tid 13214
2020-12-21 22:04:33.730 13144-13144/com.ichi2.anki I/DeckPicker$UpdateDeckListListener: Updating deck list UI
2020-12-21 22:04:33.751 13144-13213/com.ichi2.anki D/AnkiStatsTaskHandler$DeckPreviewStatistics: Starting DeckPreviewStatistics
2020-12-21 22:04:33.752 13144-13144/com.ichi2.anki D/DeckPicker$UpdateDeckListListener: Startup - Deck List UI Completed
2020-12-21 22:04:33.752 13144-13144/com.ichi2.anki D/CollectionTask: enabling garbage collection of mPreviousTask...
2020-12-21 22:04:33.754 13144-13214/com.ichi2.anki D/HostConnection: HostComposition ext ANDROID_EMU_CHECKSUM_HELPER_v1 ANDROID_EMU_dma_v1 ANDROID_EMU_YUV420_888_to_NV21 ANDROID_EMU_YUV_Cache ANDROID_EMU_async_unmap_buffer GL_OES_vertex_array_object GL_KHR_texture_compression_astc_ldr ANDROID_EMU_gles_max_version_2 
2020-12-21 22:04:33.754 13144-13214/com.ichi2.anki E/eglCodecCommon: GoldfishAddressSpaceHostMemoryAllocator: ioctl_ping failed for device_type=5, ret=-1
2020-12-21 22:04:33.764 13144-13213/com.ichi2.anki D/AnkiStatsTaskHandler$DeckPreviewStatistics: DeckPreviewStatistics query: select count(), sum(time)/1000 from revlog where id > 1608523200000
2020-12-21 22:04:33.766 13144-13144/com.ichi2.anki D/CollectionTask: enabling garbage collection of mPreviousTask...
2020-12-21 22:04:33.769 13144-13217/com.ichi2.anki W/cr_CrashFileManager: /data/user/0/com.ichi2.anki/cache/WebView/Crash Reports does not exist or is not a directory
2020-12-21 22:04:33.787 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/FontFamily;-><init>()V (light greylist, reflection)
2020-12-21 22:04:33.787 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/FontFamily;->addFontFromAssetManager(Landroid/content/res/AssetManager;Ljava/lang/String;IZIII[Landroid/graphics/fonts/FontVariationAxis;)Z (light greylist, reflection)
2020-12-21 22:04:33.787 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/FontFamily;->addFontFromBuffer(Ljava/nio/ByteBuffer;I[Landroid/graphics/fonts/FontVariationAxis;II)Z (light greylist, reflection)
2020-12-21 22:04:33.787 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/FontFamily;->freeze()Z (light greylist, reflection)
2020-12-21 22:04:33.787 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/FontFamily;->abortCreation()V (light greylist, reflection)
2020-12-21 22:04:33.787 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/graphics/Typeface;->createFromFamiliesWithDefault([Landroid/graphics/FontFamily;Ljava/lang/String;II)Landroid/graphics/Typeface; (light greylist, reflection)
2020-12-21 22:04:33.991 13144-13214/com.ichi2.anki D/EGL_emulation: eglMakeCurrent: 0xe36b4f40: ver 2 0 (tinfo 0xd1047880)
2020-12-21 22:04:34.091 13144-13144/com.ichi2.anki D/DeckPicker: onCreateOptionsMenu()
2020-12-21 22:04:34.098 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/widget/AutoCompleteTextView;->doBeforeTextChanged()V (light greylist, reflection)
2020-12-21 22:04:34.098 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/widget/AutoCompleteTextView;->doAfterTextChanged()V (light greylist, reflection)
2020-12-21 22:04:34.098 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden method Landroid/widget/AutoCompleteTextView;->ensureImeVisible(Z)V (light greylist, reflection)
2020-12-21 22:04:34.131 13144-13144/com.ichi2.anki D/BadgeDrawableBuilder: Adding badge
2020-12-21 22:04:34.141 13144-13144/com.ichi2.anki D/Collection: undoAvailable() undo size: 0
2020-12-21 22:04:34.165 13144-13144/com.ichi2.anki W/com.ichi2.anki: Accessing hidden field Landroid/view/View;->mAccessibilityDelegate:Landroid/view/View$AccessibilityDelegate; (light greylist, reflection)

**Comment by mikehardy** [comment]
Please use triple-backticks to enclose text like that, it really helps :-). But! Thank you for the stack trace that helps

**Comment by Sunt-ing** [comment]
```
2020-12-21 22:06:25.589 13144-13144/com.ichi2.anki E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.ichi2.anki, PID: 13144
    java.lang.RuntimeException
        at com.ichi2.anki.ModelBrowser$LoadingModelsHandler.actualOnPostExecute(ModelBrowser.java:119)
        at com.ichi2.anki.ModelBrowser$LoadingModelsHandler.actualOnPostExecute(ModelBrowser.java:101)
        at com.ichi2.async.TaskListenerWithContext.onPostExecute(TaskListenerWithContext.java:50)
        at com.ichi2.async.CollectionTask.onPostExecute(CollectionTask.java:250)
        at android.os.AsyncTask.finish(AsyncTask.java:695)
        at android.os.AsyncTask.access$600(AsyncTask.java:180)
        at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:712)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2020-12-21 22:06:25.590 13144-13144/com.ichi2.anki D/UsageAnalytics: sendAnalyticsException() description/fatal: java.lang.RuntimeException/true
2020-12-21 22:06:25.591 13144-13144/com.ichi2.anki E/ACRA: ACRA caught a RuntimeException for com.ichi2.anki
    java.lang.RuntimeException
        at com.ichi2.anki.ModelBrowser$LoadingModelsHandler.actualOnPostExecute(ModelBrowser.java:119)
        at com.ichi2.anki.ModelBrowser$LoadingModelsHandler.actualOnPostExecute(ModelBrowser.java:101)
        at com.ichi2.async.TaskListenerWithContext.onPostExecute(TaskListenerWithContext.java:50)
        at com.ichi2.async.CollectionTask.onPostExecute(CollectionTask.java:250)
        at android.os.AsyncTask.finish(AsyncTask.java:695)
        at android.os.AsyncTask.access$600(AsyncTask.java:180)
        at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:712)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2020-12-21 22:06:25.677 13144-13239/com.ichi2.anki W/VideoCapabilities: Unrecognized profile 4 for video/hevc
2020-12-21 22:06:25.696 13144-13239/com.ichi2.anki I/VideoCapabilities: Unsupported profile 4 for video/mp4v-es
2020-12-21 22:06:25.737 13144-13248/com.ichi2.anki E/AcraAnalyticsInteraction: ACRA handling crash, sending analytics exception report
2020-12-21 22:06:25.739 13144-13248/com.ichi2.anki D/UsageAnalytics: sendAnalyticsEvent() category/action/value/label: ACRA Crash Handler/UUID 741efefa-e3a5-4eef-a2e8-e5032cead758/-2147483648/null
2020-12-21 22:06:25.739 13144-13248/com.ichi2.anki D/UsageAnalytics: getOptIn() status: true
2020-12-21 22:06:25.805 13144-13214/com.ichi2.anki D/EGL_emulation: eglMakeCurrent: 0xe36b4f40: ver 2 0 (tinfo 0xd1047880)
2020-12-21 22:06:27.756 13144-13144/com.ichi2.anki I/Process: Sending signal. PID: 13144 SIG: 9
```

**Comment by mikehardy** [comment]
Do you have a collection.apkg you could export (and change filename to .zip for upload here) that you could share so we could reproduce?

**Comment by Sunt-ing** [comment]
[collection-20201221222207.zip](https://github.com/ankidroid/Anki-Android/files/5724807/collection-20201221222207.zip)

**Comment by Arthur-Milchior** [comment]
I can reproduce. I'll take a look at it

**Comment by mikehardy** [comment]
@Sunt-ing your help was vital in finding and fixing this issue thank you! I will have a new alpha out shortly with the fix, but it is already merged into the main branch

**Comment by Sunt-ing** [comment]
Oh, that's great!

**Comment by Sunt-ing** [comment]
> @Sunt-ing your help was vital in finding and fixing this issue thank you! I will have a new alpha out shortly with the fix, but it is already merged into the main branch

Oh, that's great!
